docs-content/operator-guide.html.md.erb

```diff
@@ -111,6 +111,17 @@ For example if 6 instances are configured with each instance having 200G, total
   <li>Record the IP address of the load-balancer. This needs to be entered in the TAS for VMs tile while configuring the "External S3 Compatible blobstore" section.</li>
   </ul>
 
+<li>Configure File Storage in your Application Service Tile<br/>
+  <ul>
+  <li>Under "Configure your Cloud Controller's filesystem" select "External S3-compatible filestore"</li>
+  <li>For "URL endpoint", provide the previously recorded load-balancer IP address in the form of "http://<IP address>"</li>
+  <li>The "Access Key" and "Secret Key" are the previously recoreded AccessKey and SecretKey.</li>
+  <li>The "S3 signature version" should be set to "V2 Signature"</li>
+  <li>The "Region" should be set to "us-east-1".</br><i>Note: Even though Minio has no concept of regions, it maintains this interface to be compatible with AWS S3 and is required to be filled in.</i></li>
+  <li>The Buildpacks, Droplets, Packages, and Resources bucket names should match the ones from the "Buckets" pane in the Minio tile.</li>
+  <li><strong>Warning:</strong> Minio currently does not support versioned backup, if you'd like to setup backups, proceed to the Enable Blobstore Backups Section.</li>
+  </ul>
```

Comment:
@XenoPhex thanks for this documentation. Can you align the above steps with the latest documentation. The section `12. Note down the MinIO Internal Blobstore access details` covers these steps.

Reply:
I'll have time later this week to clean this up.